content/docs/command-reference/remote/modify.md

```diff
@@ -359,10 +360,14 @@ more information.
   with the service account. Optional when `gdrive_use_service_account` is on.
 
   ```dvc
-  $ dvc remote modify myremote \
+  $ dvc remote modify --local myremote \
                       gdrive_service_account_user_email <user email>
   ```
 
+> The email addresses and `.p12` file path (may) contain sensitive user info.
```

Comment:
please don't forget to remove extra notes if they are not needed

Reply:
All gone.